test/Form_test.js

```diff
@@ -190,8 +190,15 @@ describe("Form", () => {
   });
 
   describe("Custom submit buttons", () => {
-    it("should submit the form when clicked", () => {
-      const onSubmit = sandbox.spy();
+    it("should submit the form when clicked", done => {
```

Comment:
Why did you have to change this ?

Reply:
Probably because calling `onSubmit` now happens asynchronously.